### ISSUE TITLE
Does not give the same output as `utmpdump`

### ISSUE THREAD

**Issue by MarcelRobitaille** [upsuper/utmp-rs#1]
First off, thank you so much for this package. I've really been looking for something like this.

My issue is that `utmpdump /var/log/wtmp` has entries from today, as does `last`, but the latest line from your package is from Nov. 8.

```
$ utmpdump /var/log/wtmp
...
[7] [01831] [:0  ] [marcel  ] [tty7        ] [:0                  ] [0.0.0.0        ] [2021-11-19T19:44:22,546916+00:00]
```

Last entry from your package:
```
UserProcess { pid: 1370446, line: "pts/80", user: "marcel", host: "127.0.0.1", session: 0, time: 2021-11-08T20:43:38.184625Z }
```

Here is the code I am using:
```rust
let entries = utmp_rs::parse_from_path("/var/log/wtmp")?;
let mut entries = entries
    .iter()
    .filter(|x| matches!(x, UtmpEntry::UserProcess { .. }))
    .collect::<Vec<_>>();
for entry in entries {
    println!("{:?}", entry);
}

```

**Comment by upsuper** [comment]
Thanks for reporting the issue.

I tried the code with `wtmp` on my own machine, and there doesn't seem to be a problem. If you don't mind, could you send over your `wtmp` file (to my email `github [at] upsuper.org`) so that I can take a closer look into the issue?

**Comment by MarcelRobitaille** [comment]
Sent.

**Comment by upsuper** [comment]
Thanks for your help! This was a silly mistake in my code :sweat_smile:

**Comment by upsuper** [comment]
I've published a new version 0.1.2 which should fix the issue.

**Comment by MarcelRobitaille** [comment]
Awesome, thank you for resolving this so quickly.
